src/lib/index.js

```diff
@@ -484,6 +484,27 @@ lib.setScale = function(element, x, y) {
     return transform;
 };
 
+lib.setPointScale = function(selection, x, y) {
```

Comment:
This is scatter-specific. It's here since related functions are right next to it, but it probably should not in `lib/`.